OrdersApplication/ItemsCell.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  ItemsCell.swift
+//  OrdersApplication
+//
+//  Created by Esraa Khaled   on 24/09/2022.
+//
+
+import UIKit
+
+class ItemsCell: UITableViewCell {
+
+    @IBOutlet weak var parentView: UIView!
+    @IBOutlet weak var itemImage: UIImageView!
+    @IBOutlet weak var quantityValue: UILabel!
+    @IBOutlet weak var unitValue: UILabel!
+    @IBOutlet weak var priceValue: UILabel!
+    @IBOutlet weak var view: UIView!
+    override func awakeFromNib() {
+        super.awakeFromNib()
+       
+        view.layer.borderWidth = 0.5
+        view.layer.borderColor = UIColor.borderPrimary?.cgColor
+        parentView.layer.cornerRadius = 5.0
+        parentView.layer.masksToBounds = true
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+
+        
+    }
+    
+}
```

Comment:
```suggestion
final class ItemsCell: UITableViewCell {

    // MARK: Outlets
    
    @IBOutlet private(set) weak var parentView: UIView!
    @IBOutlet private(set) weak var itemImage: UIImageView!
    @IBOutlet private(set) weak var quantityValue: UILabel!
    @IBOutlet private(set) weak var unitValue: UILabel!
    @IBOutlet private(set) weak var priceValue: UILabel!
    @IBOutlet private(set) weak var view: UIView!
    
    // MARK: Lifecycle
    
    override func awakeFromNib() {
        super.awakeFromNib()
       
        view.layer.borderWidth = 0.5
        view.layer.borderColor = UIColor.borderPrimary?.cgColor
        parentView.layer.cornerRadius = 5.0
        parentView.layer.masksToBounds = true
    }
}
```
ًwe shouldn't override empty method